src/discljord/permissions.clj

```diff
@@ -35,10 +36,21 @@
    :manage-webooks 0x20000000
    :manage-emojis 0x40000000})
 
+(def permissions-key
+  "Map from binary flag representation to permission name."
+  (map-invert permissions-bit))
+
+(defn permission-flags
+  "Returns a sequence of all permissions included in a given permission integer."
+  [perms-int]
+  (->> (vals permissions-bit)
+       (filter (comp (complement zero?) (partial bit-and perms-int)))
+       (map permissions-key)))
```

Comment:
For consistency, the intended type of any permissions groups is a set, and I think this function should also return a set. This will become much more clear when I go back in and spec everything.